python/pipeline/preprocess.py

```diff
@@ -88,6 +89,13 @@ class Galvo(dj.Part):
         raster_phase            : float             # shift of odd vs even raster lines
         """
 
+        def get_fix_raster(self):
```

Comment:
Please add a simple docstring explaining what this does. Something along the line of `returns a function that performs raster correction on the image`.